romanisim/nonlinearity.py

```diff
@@ -74,76 +78,20 @@ def repair_coefficients(coeffs):
         linear values with slopes of unity.
     """
     res = coeffs.copy()
+
+    if inverse:
```

Comment:
I think you're right that we can just get rid of inverse heal and elsewhere.  I think I was imagining the NL object having both the inverse and forward coefficients in it, so there would be one linearity object that knew how to reverse itself.  But your design is also good.

Reply:
removed.